tests/ethereum/ethereum.spec.ts

```diff
@@ -0,0 +1,642 @@
+              await hre.deployments.get(STABLE_DEBT_TOKEN_IMPL_ID)
+            ).address,
+            variableDebtTokenImpl: (
+              await hre.deployments.get(VARIABLE_DEBT_TOKEN_IMPL_ID)
+            ).address,
+            underlyingAssetDecimals: 18,
+            interestRateStrategyAddress: rateStrategy.address,
+            underlyingAsset: WRAPPED_NATIVE_TOKEN_PER_NETWORK["main"],
+            treasury: (await hre.deployments.get(TREASURY_PROXY_ID)).address,
+            incentivesController: (
+              await hre.deployments.get(INCENTIVES_PROXY_ID)
+            ).address,
+            aTokenName: "WETH Ethereum AToken",
+            aTokenSymbol: "aEthWETH",
+            variableDebtTokenName: "WETH Ethereum Variable Debt Token",
+            variableDebtTokenSymbol: "vDebtEthWETH",
+            stableDebtTokenName: "WETH Ethereum Stable Debt Token",
+            stableDebtTokenSymbol: "WETH Ethereum AToken",
+            params: "0x",
+          },
+        ])
+      );
+
+      // Config WETH risk params
+      const payload = {
+        asset: WRAPPED_NATIVE_TOKEN_PER_NETWORK["main"],
+        baseLTV: strategyWETH.baseLTVAsCollateral,
+        liquidationThreshold: strategyWETH.liquidationThreshold,
+        liquidationBonus: strategyWETH.liquidationBonus,
+        reserveFactor: strategyWETH.reserveFactor,
+        borrowCap: strategyWETH.borrowCap,
+        supplyCap: strategyWETH.supplyCap,
+        stableBorrowingEnabled: strategyWETH.stableBorrowRateEnabled,
+        borrowingEnabled: strategyWETH.borrowingEnabled,
+        flashLoanEnabled: strategyWETH.flashLoanEnabled,
+      };
+      await waitForTx(
+        await poolConfigurator.configureReserveAsCollateral(
+          payload.asset,
+          payload.baseLTV,
+          payload.liquidationThreshold,
+          payload.liquidationBonus
+        )
+      );
+      await waitForTx(
+        await poolConfigurator.setReserveBorrowing(
+          payload.asset,
+          payload.borrowingEnabled
+        )
+      );
+      await waitForTx(
+        await poolConfigurator.setBorrowCap(payload.asset, payload.borrowCap)
+      );
+      await waitForTx(
+        await poolConfigurator.setReserveStableRateBorrowing(
+          payload.asset,
+          payload.stableBorrowingEnabled
+        )
+      );
+      await waitForTx(
+        await poolConfigurator.setSupplyCap(payload.asset, payload.supplyCap)
+      );
+      await waitForTx(
+        await poolConfigurator.setReserveFactor(
+          payload.asset,
+          payload.reserveFactor
+        )
+      );
+      await waitForTx(
+        await poolConfigurator.setReserveFlashLoaning(payload.asset, true)
+      );
+      const aaveOracle = (await getAaveOracle()).connect(
+        impersonatedExecutor.signer
+      );
+      await waitForTx(
+        await aaveOracle.setAssetSources(
+          [payload.asset],
+          ["0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"]
```

Comment:
not sure if we want something like `const CHAINLINK_ETHUSD_AGGREGATOR_PROXY = "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419" for readbility
`

Reply:
or just a comment //